roles/nuage_master/tasks/certificates.yml

```diff
@@ -10,7 +10,7 @@
 
 - name: Create the req file
   command: >
-    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ ansible_nodename }}"
+    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ openshift_master_cluster_hostname }}"
```

Comment:
In our documentation we were planning to mandate the setting of the openshift_master_cluster_hostname for both HA and non-HA mode. In case of HA it would be the load balancer host while in non-HA it would be the single master hostname.

But I would do what is right. I am still not clear what variable should I use ?